app/controllers/rejoin/StartRejoinJourneyController.scala

```diff
@@ -53,11 +53,17 @@ class StartRejoinJourneyController @Inject()(
       (for {
         registrationWrapperResponse <- registrationConnector.getRegistration()
         currentReturnsResponse <- returnStatusConnector.getCurrentReturns(request.iossNumber)
+        vatCustomerInfoResponse <- registrationConnector.getVatCustomerInfo()
```

Comment:
`registrationWrapperResponse` already has vatinfo in it, so you shouldn't need to do a separate call to get the vat customer info

Reply:
Changes made